src/index.js

```diff
@@ -1,29 +1,46 @@
 import './style.css';
+import todoStatusUpdate from './todoStatusUpdate.js';
+import dragDrop from './dragDrop.js';
+
+const tasks = localStorage.getItem('items')
+  ? JSON.parse(localStorage.getItem('items'))
+  : [{ index: 0, description: 'Learn webpack', completed: false }, { index: 1, description: 'Read 20 pages of book', completed: false }, { index: 2, description: 'Create portfolio', completed: false }];
+
+localStorage.setItem('items', JSON.stringify(tasks));
+const data = JSON.parse(localStorage.getItem('items'));
+
+const createList = (todoItem) => {
```

Comment:
- [ ] It looks like a createList function doesn't have a sorting functionality, when I drag and drop an element, all elements should be listed in order according to an id, this will help to change the order and remember the position for each one hen you use `local storage` to remember that position if I refresh the page. I kindly invite you to add id's for elements and this has a sorting functionality when the element is dragged according to the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=one%20will%20contain%20methods%20related%20to%20drag%20/%20drop%20and%20sorting%20functionality.). 🐶